llvm/test/CodeGen/DirectX/ContainerData/RootSignature-MultipleEntryFunctions-Error.ll

```diff
@@ -0,0 +1,27 @@
+; RUN: not llc %s --filetype=obj -o - 2>&1 | FileCheck %s
```

Comment:
This test crashes in DXContainerGlobals in a way that's unrelated to your pass, which is probably why you were having the issues that led you to add `report_fatal_error`.

It would be easier to write this test using `opt`, like `not opt -passes='print<dxil-root-signature>' %s -S -o - 2>&1 | FileCheck %s` - see my overall comment for more on that.